extension/getRootCategories.js

```diff
@@ -0,0 +1,98 @@
+const fetch = require('node-fetch')
+const RootCategories = require('./models/catalog/rootCategories/rootCategories')
+let Shopify = null
+
+/**
+ * @param context
+ * @param input
+ * @param cb
```

Comment:
Change this to not be an async function anymore. So callbacks in this function are allowed

Reply:
Since this is not implemented within the current sgconnect version, i'll keep this

Reply:
It is fully supported by our servers. This is also fully supported starting from platform-sdk version 1.3.0.

It doesn't really make sense to keep on working with cb as it was done prior to the new developemnt.
This will therefore need to change, but I guess it's okay, if the other team takes care of that, as we discussed.